webapps/plf-meeds-extension/src/main/webapp/WEB-INF/conf/meeds/portal/meeds/portal.xml

```diff
@@ -28,4 +28,6 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <portal-layout>
        <page-body> </page-body>
     </portal-layout>
+    <displayed>true</displayed>
+    <display-order>1</display-order>
```

Comment:
May be change this to be displayed in top XML definition as commented in portal PR.